apps/main/src/dex/hooks/usePoolAlert.tsx

```diff
@@ -66,9 +66,13 @@ class CrossCurveLabel {
   }
 }
 
-const usePoolAlert = (poolAddress: string | undefined, hasVyperVulnerability: boolean | undefined) => {
+const usePoolAlert = (poolData?: PoolData | PoolDataCache) => {
   const params = useParams() as UrlParams
 
+  const poolAddress = poolData?.pool.address
+  const hasVyperVulnerability = poolData?.hasVyperVulnerability
+  const tokens = poolData?.tokenAddresses ?? []
```

Comment:
Can we give proper names to these tokens? [0] and [1] isn't really readable
```suggestion
  const [tokenWithName1, tokenWithName2] = poolData?.tokenAddresses ?? []
```